package.json

```diff
@@ -166,14 +167,15 @@
     "preanalyze-bundles": "cross-env WEBPACK_OUTPUT_JSON=1 CALYPSO_ENV=production NODE_ARGS=\"--max_old_space_size=8192\" npm run -s build",
     "analyze-bundles": "webpack-bundle-analyzer stats.json public -p 9898",
     "autoprefixer": "postcss -r --use autoprefixer --autoprefixer.browsers \"last 2 versions, > 1%, Safari >= 8, iOS >= 8, Firefox ESR, Opera 12.1\"",
```

Comment:
I left it in there in case one of the CSS steps e.g. editor wants to back out of using any further postcss plugins apart from autoprefixer. So it's more left as an option, it's not a necessity for the current build. Want me to remove it? :)